app/invocation/invocation_overview.tsx

```diff
@@ -47,14 +47,21 @@ export default class InvocationOverviewComponent extends React.Component {
     router.navigateToSetup();
   }
 
+  getOwningGroupName() {
+    const invocation = this.props.model.invocations[0];
+    if (!invocation) return null;
+
+    return this.props.user?.groups?.find((group) => group.id === invocation.acl.groupId)?.name;
```

Comment:
this won't work in the case where you're sent a link from an organization that you're not part of - but has been made public, will it?

let's say i'm doing some buildbuddy work in the buildbuddy organization, but then make my build public to post on a github issue for some open source project.

i'm fine if we don't want to expose the org name for now, but maybe we want to say "Public" or something.

Reply:
Good catch, to get the org name we'd have to make a `GetGroup` request in that case. WDYT about hiding those two parts of the breadcrumb, since they're currently bound to "handleOrgClicked"? So all the user would see is "Invocation 123-456..."

Reply:
sounds good